src/FlysystemOtherManager.php

```diff
@@ -251,7 +251,17 @@ public function __construct($app)
             });
         } elseif (class_exists('\Phlib\Flysystem\Pdo\PdoAdapter')) {
             $this->extend('pdo', function ($app, $config) {
-                return $this->createFlysystem(new \Phlib\Flysystem\Pdo\PdoAdapter(DB::connection($config['database'])->getPdo()));
+                $driverConfig = Arr::only($config, [
+					'database',
```

Comment:
Do we really need to pass `database` along, given we're already passing in a configured PDO instance pointing at it?

Reply:
This isn't actually used in the adapter itself, so I dropped it.